src/pages/News/index.js

```diff
@@ -1,7 +1,14 @@
 import newsContainer from './container/newsContainer';
```

Comment:
Your not using news container so remove the import statement

Reply:
ok done